ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -125,6 +127,13 @@ public void shouldPauseAndResumeQuery() {
     RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "PAUSE " + queryId + ";");
 
     assertThat(getPausedCount(), equalTo(1L));
+    REST_APP.getEngine().metricCollectors().getMetrics();
+    final MetricName key = REST_APP.getEngine().metricCollectors().getMetrics().metrics().keySet()
+            .stream()
+            .filter(metricName -> metricName.name().contains("ksql-query-status"))
+            .findFirst().get();
+    final KafkaMetric metric = REST_APP.getEngine().metricCollectors().getMetrics().metric(key);
+    assertThat((String)metric.metricValue(), equalTo("PAUSED"));
```

Comment:
Also check that the metric is as expected after we RESUME?

Reply:
Good idea; doing it.